Content.Server/_RMC14/Xenonids/Hive/XenoHiveSystem.cs

```diff
@@ -54,7 +66,19 @@ public override void Update(float frameTime)
                 continue;
 
             var popup = $"The Hive can now support: {string.Join(", ", _announce)}";
-            _xenoAnnounce.AnnounceSameHive(default, popup, hive.AnnounceSound, PopupType.Large);
+            // below is commented out because it used to not do anything, now it works but isnt localized so i think its best to have a dedicated pr look at it
+            //_xenoAnnounce.AnnounceTohive(hiveId, popup, hive.AnnounceSound, PopupType.Large);
         }
     }
+
+    /// <summary>
+    /// Create a new hive which gets networked to everyone for prediction.
+    /// </summary>
+    public EntityUid CreateHive(string name, EntProtoId? proto = null)
+    {
+        var ent = Spawn(proto ?? "CMXenoHive", MapCoordinates.Nullspace);
+        _metaData.SetEntityName(ent, name);
+        _pvsOverride.AddGlobalOverride(ent);
```

Comment:
What shared code needs this? This would make it so every client gets information about the Queen's death and hive tier unlocks

Reply:
i originally wanted it only overriden to the xenos in the hive but that refused to work, i doubt many people will use malf clients and even then they can know queen died on common radio when she is killed by marines

Reply:
ok made it handle changing hive not just player attach now force send works